### DISCUSSION TITLE
[Request/Idea] Non-Interactive version

### DISCUSSION THREAD

**Discussion by n3oney** [ThioJoe/YT-Spammer-Purge#78] [Ideas]
It would be great to have a non-interactive version that we would give options to and would not require any further input. That would let us use the software in bash scripts and run it as a cronjob.

**Comment by KendallDoesCoding** [comment]
> It would be great to have a non-interactive version that we would give options to and would not require any further input. That would let us use the software in bash scripts and run it as a cronjob.

Well, then how would you chose the channel/video you want to delete the scammers comments from?

**Reply by Firecul** [reply]
I'd imagine just picking a channel and maybe the last 2-to 3 videos once a day would suffice.

**Comment by ThioJoe** [comment]
This has been added for version 2.0 via the config file feature, you can try it now, I just haven't made an executable release yet as I make sure I get rid of bugs

**Reply by geerlingguy** [reply]
Do you have any docs on how to try it out?

Ah... config file is documented here: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Config-File-Settings

**Reply by KendallDoesCoding** [reply]
Dang, didn't expect you to be here! Big fan

**Reply by TechStudent10** [reply]
Oh hi Jeff. Didn't expect to see you here.

**Reply by ThioJoe** [reply]
> Do you have any docs on how to try it out?
> 
> Ah... config file is documented here: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Config-File-Settings

Yea that wiki page is a bit outdated at the moment, but only in the sense that I haven't added more recent settings to it, so anything on there should still be correct.

The default config file might be more useful because I explain all the settings in that anyway, and in more detail: https://github.com/ThioJoe/YT-Spammer-Purge/blob/main/assets/default_config.ini

**Reply by TechStudent10** [reply]
@ThioJoe maybe you should add a `docs` folder and add all your documentation there instead of the wiki. That would allow others to correct mistakes, update docs, etc.
